terraform/cluster_build/main.tf

```diff
@@ -198,3 +198,15 @@ module "kms" {
     "serviceAccount:${local.clu_service_account}",
   ]
 }
+
+module "acm" {
+  depends_on = [
+    module.gke,
+  ]
+  count             = var.config_sync ? 1 : 0
+  source            = "./acm/"
```

Comment:
could we rename the folder to /modules/ ? It's a better pattern in the chance we need to add more modules in the future.

Reply:
Al, do you want the ACM folder renamed to modules, or nested in a new modules folder? I think the latter would be best for future local modules. "clusterbuild/modules/ACM".